src/main/routes.ts

```diff
@@ -72,11 +72,16 @@ export class Routes {
 
     const isRouteForUser = (req: AppRequest, res: Response, next: NextFunction): void => {
       const isApp2Route = [APPLICANT_2, RESPONDENT].some(prefixUrl => req.path.includes(prefixUrl));
-      if ((isApp2Route && !req.session.isApplicant2) || (!isApp2Route && req.session.isApplicant2)) {
+      if (
+        (isApp2Route && !req.session.isApplicant2) ||
+        (!isApp2Route && req.session.isApplicant2) ||
```

Comment:
I think these two lines are basically an XOR which would be logically equivalent to:

```
if (isApp2Route != req.session.isApplicant2 || !getUserSequence(req).find(r => req.path.includes(r.url))
```

I would also probably use `.some` rather than `.find` as you want a boolean back rather than the element itself but it works either way